### PR TITLE
Release 5.3.5 release notes

### PR DESCRIPTION
**Pull request by tst-rwildes**

### CODE REVIEW

**Comment by tst-nfarah** [comment]
Let's create github web pages for this for easier review

File: docs/geedocs/5.3.5/answer/releaseNotes/relNotesGEE5_3_5.html [hunk 1]
```diff
@@ -73,9 +73,6 @@ <h1>Release notes: Open GEE 5.3.5<a class="headerlink" href="#release-notes-open
 <div class="docutils container">
 <div class="content docutils container">
 <p>Open GEE 5.3.5 is currently in development</p>
-<p class="rubric">New Features</p>
-<p><strong>New feature here</strong>.
-New feature description goes here.</p>
 <p class="rubric">Supported Platforms</p>
 <p>The Open GEE 5.3.5 release is supported on 64-bit versions of the
```
Comment (tst-nfarah) [inline docs/geedocs/5.3.5/answer/releaseNotes/relNotesGEE5_3_5.html:77]: we can remove "Open GEE 5.3.5 is currently in development"

**Review by tst-nfarah — CHANGES_REQUESTED** [review]

File: docs/geedocs/5.3.5/answer/releaseNotes/relNotesGEE5_3_5.html (hunk 1 above)
Comment (tst-rwildes) [reply docs/geedocs/5.3.5/answer/releaseNotes/relNotesGEE5_3_5.html:76]: done

File: docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_5.rst [hunk 2]
```diff
@@ -43,184 +36,76 @@ Release notes: Open GEE 5.3.5
...
+         * - 1772
+           - Geecheck reports server is not installed in dual configuration
+         * - 1768
+           - Option to not install documentation
+         * - 1749
+           - Allow users to serve WMS of 3D files from server UI
+         * - 1742
+           - Mis-configured Apache ErrorDocuments
+         * - 1735
+           - Vector packet description string retrieved by geglxinfo is incorrect
```
Comment (tst-hrajamoney) [inline docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_5.rst:54]: An extra space was added after 1717

**Review by tst-hrajamoney — APPROVED** [review]

**Review by tst-nfarah — APPROVED** [review]

**Pull request merged**